lib/ProfilePicture/ProfilePicture.js

```diff
@@ -0,0 +1,46 @@
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Img } from 'react-image';
+
+import { Loading } from '@folio/stripes/components';
```

Comment:
Since this library is part of the `@folio/stripes` collection, it can import from `@folio/stripes-components` directly instead of the path through the `stripes` package.